obspy/core/tests/test_event.py

```diff
@@ -126,7 +126,7 @@ def test_clear_method_resets_objects(self):
 
     @unittest.skipIf(not BASEMAP_VERSION, 'basemap not installed')
     @unittest.skipIf(
-        BASEMAP_VERSION >= [1, 1, 0] and MATPLOTLIB_VERSION == [3, 0, 1],
+        BASEMAP_VERSION or [] >= [1, 1, 0] and MATPLOTLIB_VERSION == [3, 0, 1],
```

Comment:
these workarounds shouldn't be needed after #2327 is merged but I guess they don't hurt either.. so blah